src/modules/vtol_att_control/tailsitter.cpp

```diff
@@ -330,3 +355,13 @@ bool Tailsitter::isFrontTransitionCompletedBase()
 
 	return transition_to_fw;
 }
+
+void Tailsitter::blendThrottleAfterFrontTransition(float scale)
+{
+	_v_att_sp->thrust_body[0] = scale * _v_att_sp->thrust_body[0] + (1.f - scale) * (-_last_thr_in_mc);
```

Comment:
@sfuhrer Maybe a comment here would help, had quite a hard time figuring out if the sign is correct, since the fw throttle is positive and the mc throttle is negative but the two variables are names similarly.

Reply:
Sure, good point, done!